### ISSUE TITLE
Setting up storage via environment variables

### ISSUE THREAD

**Issue by vectorkovacspeter** [jhuckaby/Cronicle#634]
Is it possible to setup the Storage configuration value via environment variables?

The option I like to set in this particular case is the S3 engine options:
 - accessKeyId
 - secretAccessKey
 - Bucket

**Comment by jhuckaby** [comment]
Yup, totally, see the docs here for the environment variable syntax: https://github.com/jhuckaby/Cronicle/blob/master/docs/CommandLine.md#environment-variables

So in your case, you'd want:

```
CRONICLE_Storage__AWS__credentials__accessKeyId
CRONICLE_Storage__AWS__credentials__secretAccessKey
CRONICLE_Storage__S3__params__Bucket
```
